easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
```

Comment:
Ah, ok, my bad. ^_^

We're not consistent there either though, let's fix that (in another PR).

So, good to leave it in here. And we really need to merge https://github.com/hpcugent/easybuild-easyblocks/pull/542.

```
$ grepi ^easyblock easybuild/easyconfigs/n/ncurses/*eb | wc -l
      52
$ ls easybuild/easyconfigs/n/ncurses/*eb | wc -l 
      56
```